res/enkf/active_list.py

```diff
@@ -35,6 +37,20 @@ def __init__(self):
     def getMode(self):
         return self._get_mode()
 
+    def getActiveIndexList(self):
+        """
+        Returns a list of indices corresponding to active parameters for a parameter node
+        Requires PARLY_ACTIVE as ActiveMode, else return None
+        """
+        mode = self.getMode()
+        if mode == ActiveMode.PARTLY_ACTIVE:
+            size = self._asize(0)
+            c_ptr = self._get_active_index_list()
+            index_list = [c_ptr[i] for i in range(size)]
+            return index_list
+        else:
```

Comment:
It might make sense to return an empty list if the list is `INACTIVE`? That is perhaps up for debate though.

Reply:
Will now return empty list if not PARTLY_ACTIVE. The function should not be used if ALL_ACTIVE since it does not know anything about the size of the list if ALL_ACTIVE.